boot/freeldr/freeldr/include/hwacpi.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```

Comment:
```suggestion
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```

Reply:
Note that it would be of interest to future-proof this by licensing it with MIT instead.
(Cc @DarkFire01 and co. for advice.)

Reply:
MIT or BSD would be cool

Reply:
Done!